seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
IMHO @ChadKreutzer this function needs to be typed in by the camper. I think its best we remove this. And similarly else where.